css/style.css

```diff
@@ -4,8 +4,251 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    overflow-x: hidden;
 }
 
+header.logo{
+    width: 30px;
+    height: 30px;
+    position: relative;
+    top: 30px;left: 120px;
+}
+
+.navbar{
+  color:rgb(88, 69, 69) 
+  
+   
+}
+.hamburger{
+  display: none;
+}
+
+  ul li a {
+    list-style-type: none;
+    font-size: 15px;
+    position: relative;
+    top: 0px;right: 100px;
+    text-decoration: none;
+    
+}
+  ul li a:hover
+  {
+  color: rgb(235, 106, 31);
+  }
+
+  li {
+    display: inline;
+    padding: 7px;
+    margin: 10px;
+    margin-top: 0;
+    float: right;
+    
+  }
+
+  .photo{
+    background-image: url(/img/first-background.jpg);
+    background-size:cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin: 0;
+    padding: 0;
+    width: 1500px;
+    height: 1000px;
+    align-items: center; 
+    position: relative;
+    top: 0px;left: 30px;
+    
+    
+  }
+
+.photo h3,
+h4 {
+  color: white;
+  text-align: center;
+  position: relative;
+  top: 270px;left: 200px;
```

Comment:
Remove `top` and `left` rules and add `display: flex` on the parent element: `.photo`